mp4parse/src/lib.rs

```diff
@@ -257,16 +257,16 @@ pub struct VPxConfigBox {
 }
 
 #[derive(Debug, Clone)]
-struct FLACMetadataBlock {
+pub struct FLACMetadataBlock {
     block_type: u8,
```

Comment:
Yes, except that STREAMINFO is always first. I didn't want to expose it because I thought an enum was better, but that's what semver bumps are for.